common/browser.go

```diff
@@ -538,9 +538,14 @@ func (b *Browser) Close() {
 	// command, which triggers the browser process to exit.
 	if !b.browserOpts.isRemoteBrowser {
 		var closeErr *websocket.CloseError
-		// Using a internal context here since vu context will very likely be
-		// closed.
-		err := cdpbrowser.Close().Do(cdp.WithExecutor(b.browserCtx, b.conn))
+		// Using the internal context with a timeout of 10 seconds here since
+		// 1. vu context will very likely be closed;
+		// 2. there's a chance that the process has died but the connection still
+		//    thinks it's open.
+		toCtx, toCancelCtx := context.WithTimeout(b.browserCtx, time.Second*10)
```

Comment:
What is the reason it's exactly 10 seconds?

Reply:
Feels like a good timeout. I was going to do 5 seconds, but 10 seemed safer in case the browser process was still alive but the k6 process was under heavy load. Any suggestions on what might be better?

Reply:
I worry that we would need to change this whenever a larger timeout is needed. A larger timeout can cause prematurely aborted errors from the engine team. Two hopefully possible solutions:
- Implementing our own `Context` implementation that automatically cancels when the process is dead
- Making this an environment variable

Reply:
Yep, that makes sense. Actually, maybe we can check something in the `browserProc` for clues that something has gone wrong 🤔 

I'll create another PR for this though.

Reply:
Sure, I've created an issue for it from the comment: https://github.com/grafana/xk6-browser/pull/1442